asab/api/service.py

```diff
@@ -84,9 +117,13 @@ def initialize_zookeeper(self, zoocontainer=None):
 	def _build_zookeeper_adv_data(self):
 		adv_data = {
 			'appclass': self.App.__class__.__name__,
-			'launchtime': datetime.datetime.utcfromtimestamp(self.App.LaunchTime).isoformat(),
+			'launchtime': datetime.datetime.utcfromtimestamp(self.App.LaunchTime).isoformat() + 'Z',
 			'hostname': self.App.HostName,
 		}
+
+		if len(self.AttentionRequired) > 0:
+			adv_data.update({"attention_required": self.AttentionRequired})
```

Comment:
Maybe you need to transform `self.AttentionRequired` into the array here - or push that as a dict, it should be fine.